02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,55 @@
+const { people } = require("../data");
+
+const getPeople = (req, res) => {
+    res.json(people);
+};
+
+const addPerson = (req, res) => {
+    try {        
+        const { name } = req.body || {};
+        
+        if (!name) {
+            return res.status(400).json({ success: false, message: "Please provide a name" });
+        }
+
+        const newPerson = { id: people.length + 1, name };
+        people.push(newPerson);
+
+        res.status(201).json({ success: true, data: newPerson });
+    } catch (error) {        
+        console.error("Error in POST /api/v1/people:", error);
+        res.status(500).json({ success: false, message: "An internal server error occurred." });
+    }
+};
+
+const getPersonById = (req, res) => {
+    const person = people.find(p => p.id === parseInt(req.params.id));
+    if (!person) {
+        return res.status(200).json({ success: false, message: "Person not found" });
```

Comment:
An HTTP status 200 OK doesn't match the situation.  In this case an HTTP 404 may be better suited.

Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status